core/templates/dev/head/player/exploration_player.html

```diff
@@ -105,6 +65,106 @@
     <feedback-popup></feedback-popup>
   </script>
 
+  <script type="text/ng-template" id="popover/informationCard">
+    <md-card class="md-padding">
+      <div ng-style="infoCardBackgroundImageCss" class="oppia-info-card-logo-thumbnail">
+        <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
+      </div>
+
+      <div class="oppia-info-card-content">
+        <p ng-if="explorationDescription">
+          <[explorationDescription | truncateAndCapitalize: 95]>
+        </p>
+        <p ng-if="!explorationDescription">No objective specified.</p>
+        <ul layout="row" class="card-metrics" layout-align="space-between center">
+          <li>
+            <span class="fa fa-star fa-lg"></span>
+            <[explorationRatings]>
+          </li>
+          <li>
+            <span class="fa fa-eye fa-lg"></span>
+            <[viewersCount | summarizeNonnegativeNumber]>
+          </li>
+          <li>
+            <span class="fa fa-clock-o fa-lg"></span>
+            <[getLocaleAbbreviatedDatetimeString(explorationLastUpdatedMsec)]>
+          </li>
+          <ul layout="row" layout-align="space-between center" class="oppia-info-card-exploration-contributors-profile">
+            <img src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
+            <li ng-repeat="ContributorUsername in explorationContributorUsernames| limitTo: 2"
+              tooltip="<[ContributorUsername]>" tooltip-placement="top">
+              <img class="img-circle" ng-src="/images/general/user_mint_48px.png" alt="">
+            </li>
+
+            <!--TODO(Makonda) get profile picture and profile url to make image click direct to contributor profile-->
+            
+            <li ng-if='explorationContributorUsernames.length > 2' class="oppia-contributors-more-circle"
+              tooltip-append-to-body=true tooltip="<[explorationContributorUsernames.slice(2).join()]>"
+              tooltip-placement="top">+<[explorationContributorUsernames.length - 2]>
+            </li>
+            <li ng-if='explorationContributorUsernames.length === 0'
+                tooltip="Community-editable" tooltip-placement="top">
+              <span class="fa fa-globe fa-lg oppia-info-card-community-editable-icon"></span>
+            </li>
+          </ul>
+        </ul>
+        <div class="oppia-info-card-bottom-row" layout="row">
+          <div flex="55">
+            <md-content layout="row">
+              <div class="oppia-info-card-tag-icon"><span class="fa fa-tags fa-lg"></span></div>
+              <md-chips layout-align="center center">
+                  <md-chip ng-if='explorationTags.length > 2' class="oppia-info-card-tooltip-more"
+                    tooltip-append-to-body=true tooltip="<[explorationTags.slice(2).join()]>"
+                    tooltip-placement="right">
+                    <[explorationTags.length - 2]>+ more tags
```

Comment:
I had forgotten, chips are primary inputs that you can add tags dynamically. I have overridden that property of the cursor